routes/dockerflow.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const express = require("express");
+const fs = require("fs");
+const path = require("path");
+const {version, homepage} = require("../package.json");
+
+
+const router = express.Router();
+
+let commit;
+try {
+  commit = require("git-rev-sync").short();
+} catch (err) {
+  console.error("Error in git-rev-sync: ", err);
```

Comment:
I think everything looks 👌!
If for some reason **git-rev-sync** fails, the `commit` will be undefined/null and be omitted from the version.json file.